### DISCUSSION TITLE
Laps object to Lap

### DISCUSSION THREAD

**Discussion by BiomedicLJZ** [theOehrly/Fast-F1#396] [Q&A]
Hi, i need a hand with something a little different, I'm doing a fastest vs slowest lap comparison with the following code:
```
fastest_lap=a.pick_fastest()

slowest_lap=a[a['LapTime']==a['LapTime'].max()]

```
but the slowest_lap object it's a Laps object and the other one its a Lap, i know they are almost the same but i wanted to know if there's a way to extract the single lap object when it's not with the `pick_fastest()`

**Comment by theOehrly** [comment]
> Hi, i need a hand with something a little different, I'm doing a fastest vs slowest lap comparison with the following code:
> 
> ```
> fastest_lap=a.pick_fastest()
> 
> slowest_lap=a[a['LapTime']==a['LapTime'].max()]
> ```
> 
> but the slowest_lap object it's a Laps object and the other one its a Lap, i know they are almost the same but i wanted to know if there's a way to extract the single lap object when it's not with the `pick_fastest()`

`Laps` is a subclass of a Pandas DataFrame and `Lap` a subclass of a Pandas Series. What you therefore want to do, is to get the first (and single) element of the Laps DataFrame. This can be done using `.iloc` to get the element by integer location:

``slowest_lap=a[a['LapTime']==a['LapTime'].max()].iloc[0]``
